src/platform/plugins/shared/esql/public/triggers/update_esql_query/update_esql_query_actions.ts

```diff
@@ -10,15 +10,14 @@
 import { i18n } from '@kbn/i18n';
 import type { Action } from '@kbn/ui-actions-plugin/public';
 import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+import { isActionCompatible, executeAction } from './update_esql_query_helpers';
 
-const ACTION_UPDATE_ESQL_QUERY = 'ACTION_UPDATE_ESQL_QUERY';
+export const ACTION_UPDATE_ESQL_QUERY = 'ACTION_UPDATE_ESQL_QUERY';
```

Comment:
Export `ACTION_UPDATE_ESQL_QUERY` from a seperate file, like `constants.ts`. Otherwise `update_esql_query_actions.ts` gets included in the page load bundle.

Reply:
Same